src/base/memory_hash.c

```diff
@@ -2170,7 +2170,40 @@ mht_get_hash_number (const int ht_size, const DB_VALUE * val)
 	case DB_TYPE_ENUMERATION:
 	  hashcode = mht_get_shiftmult32 (val->data.enumeration.short_val, ht_size);
 	  break;
+	case DB_TYPE_JSON:
```

Comment:
What is the purpose of including schema into json hash?

Reply:
removed it